bim_webviewer/bim_web_viewer.md

```diff
@@ -1727,6 +1727,94 @@ Model
 
 ---
 
+### Set Object Color
+
+<p class="heading-link-container"><a class="heading-link" href="#set-object-color"></a></p>
+
+```js
+setObjectColor(palettes);
+```
+
+#### Description
+
+Changes object color in selcted, default, and xray mode.
+
+#### Parameters
+
+| Field Name | Required | Type | Description |
+| - | - | - | - |
+| palettes | true | Object[] | color palettes with object ids: {palette: {default: {color: '#ff0000', opacity: 1}, xray: {color: '#00ff00', opacity: 1}, selected: {color: '#0000ff', opacity: 1}}, objectIds: [1, 2, 3]}. Must provide at least one of default, xray or selected color. If no color declared will use the our baked-in values or the color from the file for the "default" value. |
```

Comment:
How does this look in the table? Maybe it can be explained in a paragraph below Description?

Reply:
Should we just drop in the schema we defined? Maybe just drop in the JSON schema somewhere in here.